python/doc/theory/numerical_methods/nataf_transformation.rst

```diff
@@ -3,33 +3,26 @@
 Generalized Nataf Transformation
```

Comment:
I suggest to use an itemized list when describing what is an elliptical copula, the generic elliptical representative and the standard spherical representative:

![image](https://user-images.githubusercontent.com/31351465/229273964-0ce9ad63-b68a-4a8b-a277-077f7ff947ff.png)

Reply:
I suggest to use a proper RST sub-title instead of bold face symbols for the "Generalized transformation" sub-section:

![image](https://user-images.githubusercontent.com/31351465/229274045-c17d1c34-d79d-45b5-a8d2-d976f9afc79a.png)

Reply:
ok!